### PR TITLE
README: document winsqlite3

### PR DESCRIPTION
**Pull request by pratikpc**
https://blogs.windows.com/windowsdeveloper/2017/02/06/using-sqlite-databases-uwp-apps/

Makes it easier to find out about the awesome WinSQLite3 feature

The text is from the [Release notes](https://github.com/rusqlite/rusqlite/releases/tag/v0.24.1). I just replaced Windows 10 with Windows

### CODE REVIEW

**Comment by pratikpc** [comment]
Should Notes on Building section also be modified to mention the WinSQLite option??

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ features](https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-s
 * `extra_check` fail when a query passed to execute is readonly or has a column count > 0.
 * `column_decltype` provides `columns()` method for Statements and Rows; omit if linking to a version of SQLite/SQLCipher compiled with `-DSQLITE_OMIT_DECLTYPE`.
 * `collation` exposes [`sqlite3_create_collation_v2`](https://sqlite.org/c3ref/create_collation.html).
+* [`winsqlite3`](https://blogs.windows.com/windowsdeveloper/2017/02/06/using-sqlite-databases-uwp-apps/) allows linking against the SQLite present in newer versions of Windows
```
Comment (gwenn) [inline README.md:118]: The link is not strictly necessary and not related to rust programming but ok.

Comment (pratikpc) [reply README.md]: Done!

**Review by gwenn — APPROVED** [review]

**Comment by gwenn** [comment]
Thanks and sorry for the long delay.

**Pull request merged**